src/components/ClockDisplayImage.tsx

```diff
@@ -22,9 +22,8 @@ const ClockDisplayImage = () => {
           alt="Musketeers of Words logo (2018-2023)"
           width={80}
           height={64}
-          className="mt-32"
+          className={`mt-32 ${!clockImageLoaded && "opacity-0"}`}
           onLoad={() => setClockImageLoaded(true)}
-          hidden={!clockImageLoaded}
```

Comment:
Removing this line (and the corresponding ones) causes a misplacement of the "Loading…" caption.